impeller/entity/entity.cc

```diff
@@ -33,12 +33,16 @@ bool Entity::AddsToCoverage() const {
   return adds_to_coverage_;
 }
 
-std::optional<Rect> Entity::GetCoverage() const {
+std::optional<Rect> Entity::GetCoverage(bool use_cache) const {
```

Comment:
Can we just leave the cache out for now? Perhaps we can profile and add this memoization later.

Reply:
Sure, I removed the cache for now.